Manito/Manito/Screens/Detail-Wait/DetailEditViewController.swift

```diff
@@ -18,16 +19,22 @@ final class DetailEditViewController: UIViewController {
     // MARK: - property
     
     private let editMode: DetailEditView.EditMode
-    private let detailRoomRepository: DetailRoomRepository = DetailRoomRepositoryImpl()
-    private let room: RoomInfo
+    private var roomPublisher: CurrentValueSubject<CreatedRoomInfoRequestDTO, Never>
+    private let viewModel: any BaseViewModelType
     weak var detailWaitDelegate: DetailWaitViewControllerDelegate?
+    private var cancellable = Set<AnyCancellable>()
```

Comment:
p1;

저희 코드 컨벤션에 맞게 전체적으로 타입 명시 부탁드릴게요!
```suggestion
    private var cancellable: Set<AnyCancellable> = Set()
```

Reply:
엇 그러네요! 놓쳤네요 감사합니다!
DetailWaitViewController도 같이 수정했습니다!
[d100475](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/600/commits/d100475da5ad726c0d2ad87351e3d8c2199c352d)